examples/postgres/docker-compose.yaml

```diff
@@ -0,0 +1,40 @@
+version: "3.7"
+services:
+
+  proxy:
+    image: envoyproxy/envoy-dev:latest
+    volumes:
+      - ./envoy.yaml:/etc/envoy.yaml
+    networks:
+      envoymesh:
+        aliases:
+          - envoy
+    expose:
+      - "1999"
+      - "8001"
+    ports:
+      - "1999:1999"
+      - "8001:8001"
+    command: "/usr/local/bin/envoy -c /etc/envoy.yaml -l debug"
+
+  postgres:
+    image: postgres:latest
+    networks:
+      envoymesh:
+        aliases:
+          - postgres
```

Comment:
this alias definitely isnt necessary as its the same as the service name

Reply:
Fixed!